api/admin/controller.py

```diff
@@ -233,7 +264,7 @@ def edit(self, data_source, identifier):
                 data_source=staff_data_source,
                 subject_type=Subject.SIMPLIFIED_FICTION_STATUS,
                 subject_identifier=fiction_term,
-                weight=STAFF_WEIGHT * 100,
+                weight=WorkController.STAFF_WEIGHT * 100,
```

Comment:
We can come back to this later, but fiction status and genres should be edited in the same request. That way changing fiction status won't need to invalidate staff-selected genres, and we won't need to give it a higher weight.